packages/web/src/components/CheckboxField/_theme.scss

```diff
@@ -0,0 +1,22 @@
+@use '@tokens' as tokens;
+
+$spacing: tokens.$space-300;
+
+$text-offset: tokens.$space-400;
+
+$label-typography: tokens.$body-medium-text-regular;
+$label-color-default: tokens.$text-primary-default;
+$label-color-disabled: tokens.$text-primary-disabled;
+$label-required-color: tokens.$emotion-danger-default;
+
+$input-width: 18px;
+$input-unselected-color: tokens.$action-unselected-default;
+$input-selected-color: tokens.$action-selected-default;
+$input-disabled-color: tokens.$text-primary-disabled;
+$input-checked-mark: url('data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMTggMTgiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgd2lkdGg9IjE4IiBoZWlnaHQ9IjE4Ij48cGF0aCBkPSJNNy43MSAxMy4yOWEuOTk2Ljk5NiAwIDAgMS0xLjQxIDBMMi43MSA5LjdhLjk5Ni45OTYgMCAxIDEgMS40MS0xLjQxTDcgMTEuMTdsNi44OC02Ljg4YS45OTYuOTk2IDAgMSAxIDEuNDEgMS40MWwtNy41OCA3LjU5WiIgc3R5bGU9ImZpbGw6I2ZmZiIvPjwvc3ZnPg==');
```

Comment:
If we add something like this https://github.com/react-ui-org/react-ui/blob/master/src/lib/styles/tools/_svg.scss we can use shorter notation. But we can add that later.

Reply:
@adamkudrna wrote the same. But you both are suggesting adding it as a separate feature.

Reply:
I've just created issue for this.